components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -567,7 +568,142 @@ func generateVirtualService(instance *v1beta1.Notebook) (*unstructured.Unstructu
 	}
 
 	return vsvc, nil
+}
+
+func generateHTTPRoute(instance *v1beta1.Notebook) (*unstructured.Unstructured, error) {
+	name := instance.Name
+	namespace := instance.Namespace
+	prefix := fmt.Sprintf("/notebook/%s/%s", namespace, name)
+
```

Comment:
suggestion(nit): For uniformity across variables, this could be `pathPrefix` (given there is `pathRewrite` below)

Reply:
(resolved)